init.sh

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env bash
 
-cp -i src/stubs/Homestead.yaml Homestead.yaml
+if [[ -n "$1" ]]; then
```

Comment:
As someone not familiar with this whole conversation inspecting the code though, its easier to get context. Oh I pass in `--json` and get a json stub. It's a minor, but anything we can do to make the code easier to understand.